include/messages.h

```diff
@@ -5,12 +5,7 @@
 
 typedef s32 MessageID;
 
+// Prefer editing splat.yaml's msg ids section than using this directly!
 #define MESSAGE_ID(section, index) (((section << 0x10) + index))
```

Comment:
I know this isn't in-scope for this PR, but the shift in this macro should be in decimal